infra/aws/terraform/prow-build-cluster/resources/rbac/cluster-viewer.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: eks-cluster-viewer
+rules:
+  - apiGroups: ['*']
```

Comment:
Let's leave a TODO note that we might want to scope this down, e.g. to disallow access to Secrets.